cmsBuild

```diff
@@ -3629,6 +3637,17 @@ def build(opts, args, factory):
             finalString = "\n".join(["%s (%s)" % (dep.pkgName(), dep.checksum)
                                      for dep in getPackages(deps)])
             log(finalString)
+            #get the deps for all packages to be built
+            if opts.depsLog:
+                deplist = getPackages(deps)
+                deps_object = []
+                for pkgdep in deplist:
+                    ptchs = [os.path.join(cmsdist_gh_url ,cmsdist_branch ,os.path.basename(p)) for p in pkgdep.patches]
+                    docs = os.path.join(cmsdist_gh_url, cmsdist_branch, pkgdep.name,"docs.txt")
+                    deps_object.append({"pkgname":pkgdep.name, "version_short":pkgdep.realVersion, "version_long":pkgdep.version, "checksum": pkgdep.checksum, "patches":ptchs, "sources":pkgdep.sources, "docs":docs })
+
+                with open(os.path.join(opts.workDir, "DEPS", pkg.name+'_dependencies.json'), 'w') as depfile:
```

Comment:
as the file is already under a `DEPS` directory, so remove the extra `_dependencies` in the name, Just `package_name.json` is good enough.